test/tabbable.test.js

```diff
@@ -40,117 +58,125 @@ describe('tabbable', function() {
     cleanupFixtures();
   });
 
-  it('basic', function() {
-    var actual = fixture('basic').getTabbableIds();
-    var expected = [
-      'tabindex-hrefless-anchor',
-      'input',
-      'select',
-      'href-anchor',
-      'textarea',
-      'button',
-      'tabindex-div',
-      'hiddenParentVisible-button',
-    ];
-    assert.deepEqual(actual, expected);
-  });
+  [
+    { name: 'window', func: fixture },
```

Comment:
How about `getFixture` instead of `func`, since `func` is not descriptive?

Reply:
Good call, I'll go with `getFixture` then.  And, after looking at this comment, I came to the same realization you pointed out below: I'd failed to actually use this parameterized function!